helm/values.yaml

```diff
@@ -63,7 +62,7 @@ datacatalog:
   image:
     # -- Docker image for Datacatalog deployment
     repository: ghcr.io/flyteorg/datacatalog
-    tag: v0.3.6
+    tag: v0.3.4
```

Comment:
Why is this going to an older version?

Reply:
@EngHabu datacatalog v0.3.5 is broken. (see slack for more details)

Reply:
👍🏾